recipes/libheif/all/conanfile.py

```diff
@@ -39,8 +40,25 @@ class LibheifConan(ConanFile):
         "with_jpeg": False,
         "with_openjpeg": False,
         "with_openjph": False,
+        "with_openh264": False,
     }
 
+    @property
+    def _min_cppstd(self):
+        return "20" if Version(self.version) >= "1.19.0" else "11"
+
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "20": {
```

Comment:
From now on this should only be used for minimum compiler versions that do not compile with the given cppstd even if they support said flag